src/intentsui.cs

```diff
@@ -41,6 +41,8 @@ public enum INUIAddVoiceShortcutButtonStyle : ulong {
 		WhiteOutline,
 		Black,
 		BlackOutline,
+		Automatic, // how to handle the fact this 2 are available since iOS 13.0 ?
```

Comment:
You can add the iOS availability on enums and their individual values too. 

[iOS (13,0)]
Automatic,
[iOS (13,0)] 
AutomaticOutLine, 

Also just a small thing but add a comma after the AutomaticOutLine so that if anyone else ever adds something to your enum, their name won't show up in the blame and seem like they added that value. Great job! :)

Reply:
Thanks a lot ! 😄